arangod/Cluster/ClusterInfo.cpp

```diff
@@ -3134,6 +3138,13 @@ Result ClusterInfo::createCollectionsCoordinator(
             return r;
           }
         }
+      } else {
+        LOG_TOPIC("98675", WARN, Logger::CLUSTER)
+          << "Failed createCollectionsCoordinator for " << infos.size()
+          << " collections in database " << databaseName << " isNewDatabase: " << isNewDatabase
+          << " first collection name: " << infos[0].name << " result: " << res;
+        return Result(TRI_ERROR_HTTP_SERVICE_UNAVAILABLE,
+                      "A cluster backend which was required for the operation could not be reached");
```

Comment:
The failure is very generic. It is a precondition failure in the agency, which comes from any component's failure leading to missing Plan entry. It can have any number of reasons, which the customer cannot fix. The only thing we know unfortunately is that the Plan entry has changed since.

Reply:
It seems to me that reporting that the collection could not be finished because of a precondition failure in the agency is only confusing.

Reply:
The log output gives some specificity.